### DISCUSSION TITLE
Sobre trabalhar em startups

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#832] [Discussão]
Bom dia amgs,

Como muitos devem saber, estou trabalhando no [Cubo](https://cubo.network/) e nosso foco é impulsionar o empreendedorismo e especificamente o mercado de startups no Brasil.

O foco do meu time são nos produtos digitais do Cubo, entre eles uma plataforma que tem como objetivo, [levar um pouco do que acontece no Cubo fisico para o mundo digital](https://cubo.network/digital). Sendo assim, comecei a me interessar pelo mercado de startups no Brasil, e acompanho diariamente as dores e amores que os profissionais que atuam nesse meio vivem no dia-a-dia.

Resolvi portanto levantar uma discussão aqui:

- Você trabalha ou já trabalhou em startups? Qual?
- Caso nunca tenha trabalhado, você trabalharia?
- Caso tenha trabalhado, quais os pontos positivos e negativos?

Nóis!

**Comment by danlaurent** [comment]
Já trabalhei em startup (minha última experiência foi a primeira startup em que trabalhei), e trabalharia denovo em outra caso surgisse uma oportunidade interessante.

Os pontos positivos eram:
O ambiente, descontraído e horizontal, conseguia discutir os assuntos diretamente com o chefe sem muita burocracia como em uma empresa maior que eu já tinha passado.
Produto novo, quando a startup está bem no início, é bem bacana ver o que você está desenvolvendo tomando uma forma maior.
Se você for um dev júnior, o desenvolvimento é animal, pois geralmente a equipe é pequena e você participa de tudo um pouco.

Os pontos negativos:
As vezes você pode não se adaptar ao ritmo, a gestão tem que ser muito boa para não gerar dor de cabeça aos devs.
Em alguns casos não passa tanta segurança quanto trabalhar em uma empresa grande, mas acredito que isso não seja tãão relevante. :)

Bom, está aí minha opinião baseada na experiência que tive. :)

**Comment by FranklinSilva** [comment]
Eu sempre trabalhei em startups, desde estágios aqui e fora quando fiz intercâmbio. 
Estou na mesma startup há quase 2 anos e tenho q sempre lidar com os altos e baixos da empresa.

Somos uma plataforma de educação que foca na criação de apps de nicho através de um sistema de White label. Conseguimos servir e-books, vídeo, cursos e avaliações dentro do App q tem uns 4 anos de vida.

As vantagens são todas aquelas que aparecem nas vagas de empregos, acredito q vocês já tem uma ideia. Porem o que mais eu vejo como vantagem é a liberdade tentar novas tecnologias e me reinventar.

Quanto a desvantagens eu não tenho nada a dizer com relação a nossa área e sim com relação a como são feitos negócios. 
Já tivemos cliente q sumiu, que deu calote, que quer o sistema no ar mas não paga com nenhuma das suas obrigações, já modificamos nosso modelo de negócio 3 vezes para atender a demanda de mercado (como implementar mudanças de software na velocidade q o mercado precisa se vc tem recursos limitados?), O salário as vezes atrasa, afinal, não somos uma startup que se paga 100%.

Tem também os aspectos desafiadores de se trabalhar em uma startup. É chamar no peito funções q você não sabe, códigos que foram mau escritos/não foram escritos por vc e dizer, eu vou conseguir entregar!
isso é massa!

**Comment by MatheusGodinho** [comment]
Trabalho na LegalLabs, trabalhamos com inteligência artificial aplicada ao direito em dois ramos: Legal Strategy e geração automatizada de petições. E estamos estudando bastante novas aplicação na area jurídica, porque somos bem próximo da academia.

Os pontos positivos: equipe muito jovem, ambiente muito bom, poder desde o começo tomar decisões importantes de arquitetura e estar próximo de decisões sobre o produto mesmo sendo um dev. Trabalhar com tecnologias novas: Node + Vue.js.

Os pontos negativos: sobra muita responsabilidade em cima dos devs, salário não muito competitivo, clima de instabilidade e pouco tempo "na empresa" pra desenvolvimento da equipe.

**Comment by caioincau** [comment]
Trabalhei e não foi uma experiência legal:

-Tinhamos falta de investidores e com isso, além de estarmos sobre carregados(TI especialmente), viviamos com medo da empresa fechar e acordar sem trabalho.

- A horizontalidade, apesar de boa, fazia com que muitas demandas entrassem de forma atravessada.

- Prazos absurdos para entregar features, pois assim poderiam ser mostradas a investidores.

Lado bom:

- Chance de aprendizado enorme.

- Ambiente fisico legal(NERF/Nespresso/Snacks)

- Seu código faz a diferença, poucas pessoas codando, então tudo que você faz já impacta diretamente a empresa(positiva ou negativamente).

**Comment by jaffarc** [comment]
Atualmente trabalho em 2 startups. São projetos diferentes, mas o mesmo lugar.
Eu nunca tinha trabalho antes em startups. Essa é a primeira vez e gostei muito de trabalho aqui. 
Trabalhei em grandes empresas/projetos. Aqui não e como algumas startups que vejo em sampa, estamos em sorocaba/sp. Fazemos nosso hardware e soft.

As vantagens que vejo aqui, são que temos 2 tipos de contratação uma bolsa de fapesp e outra pj. Aqui não tem gerente de projetos ou lider, não tem ninguem na sua nuca perguntando se ja acabou. Horario vc quem faz. "Quer trabalhar no feriado e não vim na segunda tudo bem :)"

Os pontos negativos: De trabalha aqui não tenho nada, para falar e a melhor empresa que ja trabalhei. O que é ruim, vejo que fazemos muitas coisas legais aqui e como  estamos no interior,  muitas vezes não somos conhecidos. E tambem para  contratar pessoas tambem boas.

As startups são : 
http://onesports.com.br monitoramento de atletas e mostra online performace dos athetas trabalhamos com grandes times.

http://smartoys.com.br que trabalho com um projeto de lego voltado para escola aonde tem uma plataforma online que os pais acompanha o que o filho fez na escola os professores e alunos tem um aplicativos que sobe as aulas com foto video de atividade.


ops: Desculpa a acentuacao meu linux nao tem kkkkk "nao configurei..'

**Comment by renatoagds** [comment]
Eu trabalhei em 2 startups. Atualmente trabalho na Loggi, antes disso passei pelo Yahoo Brasil.
Vivi a experiência dos 2 mundos, entre uma startup vs uma empresa já consolidada.

Acho que no geral o ponto bom que retirei de trabalhar em uma startup, é o aprendizado, você tem a oportunidade de se desenvolver em outras linguagens, explorar outras áreas, com isso você evoluir em pontos fora da sua zona de conforto, o que acho super válido, e foi um dos motivos pelo qual resolvi entrar em uma startup.

O meu ponto negativo de startup, é a incerteza sobre o futuro ser relativamente maior, algo que você vai gerenciando com o tempo, porém no começo é impactante para quem vem de uma empresa mais consolidada, outro ponto é conseguir passar pelos altos e baixos da empresa, como qualquer startup, vai rolar decisões erradas, muitas vezes em nível gerencial, e conseguir superar esses obstáculos pode ser bem complicado.

No geral gosto bastante, e pretendo continuar, pra alcançar meu objetivo de participar do crescimento de uma empresa, para quem quiser conhecer, essa é a Loggi:

https://www.loggi.com/

**Comment by yanccprogramador** [comment]
Atualmente trabalho na 4yousee digital signage, atualmente o mercado que melhor me adaptei. Tem pouco tempo mas eu posso citar que, como vantagens, na área de TI uma startup tem um clima mais descontraído, tem também pouca hierarquia é mais Horizontal coisas que eu vejo como vantagens, além de grande conhecimento e dificuldades que nos ajuda a ser melhor profissional, porém no outro lado temos que em startups pequenas muitas vezes ficamos muito reféns do cliente fazendo coisas que não seria ideal apenas para não perder o cliente isso é uma desvantagem

**Comment by StanleySathler** [comment]
Tive a oportunidade de trabalhar remotamente para duas startups estrangeiras: a Language Zen, EUA; e Moteefe, Inglaterra (onde trabalho atualmente).

**Vantagens**

A maior vantagem, acredito eu, é o sentimento de que você é uma peça muito mais importante no tabuleiro. Geralmente com poucos funcionários e um modelo horizontal, onde você tem contato direto com seu CTO o tempo todo, você tem a sensação de ser muito mais importante, de afetar muito mais no crescimento da empresa.

Também percebi que a chance de aprendizado é muito maior. Quando trabalhei em empresas maiores, dificilmente eu tocava no Jenkins ou mesmo fazia o deploy, por exemplo. Em startups, você tem a chance de lidar mais diretamente com o tooling. Fora que em startups, é mais comum você encontrar e trabalhar com tecnologias mais emergentes.

**Desvantagens**

Como desvantagem eu citaria o salário geralmente mais baixo. No caso de empresas estrangeiras, a diferença de moeda acaba sendo uma vantagem para ambos os lados: quem recebe e quem paga.

Como disseram, alguns prazos podem ser bem curtos também. Não apenas pelas apresentações aos investidores, mas pela necessidade de agradar os usuários que às vezes estão reclamando muito de alguma coisa. E sabemos que no período inicial, perder usuário é o que não queremos.
